back/engines/commercial/public_api/app/policies/public_api/phase_policy.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module PublicApi
+  class PhasePolicy < PublicApiPolicy
+    class Scope
+      attr_reader :api_client, :scope
```

Comment:
These attribute readers are only used in this class, so they can be moved to the `private` section.